src/app.js

```diff
@@ -1,21 +1,130 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
+const fs = require('fs');
+const path = require('path');
+
+const terminal = process.argv.slice(2);
+
+function renameFile() {
+  const pathFile = path.join(__dirname, terminal[0]);
+
+  if (/.txt/.test(pathFile)) {
+    const pathInfo = path.join(__dirname, terminal[1]);
+
+    fs.readFile(pathFile, (err, data) => {
+      if (err) {
+        process.stdout.write('This terminal[0] could not be read');
+
+        return;
+      }
+
+      fs.writeFile(pathInfo, data, (error) => {
+        if (error) {
+          process.stdout.write('Error creating new terminal[0]');
```

Comment:
Error messages should be more descriptive and helpful. Instead of 'Error creating new terminal[0]', consider something like `Error while writing to the destination file: ${pathInfo}`